azure-pipelines/loc.yml

```diff
@@ -32,10 +32,10 @@ variables:
 - group: OneLocBuildVariables
 
 extends:
-  template: v1/1ES.Unofficial.PipelineTemplate.yml@1ESPipelineTemplates
+  template: v1/1ES.Official.PipelineTemplate.yml@1ESPipelineTemplates
   parameters:
     pool:
-      name: NetCore1ESPool-Svc-Internal
+      name: netcore1espool-internal
```

Comment:
Is this case-sensitive? Might be worthy to test

Reply:
I just copied it from the azure-pipelines-official.yml